main.tex

```diff
@@ -1137,31 +1240,31 @@ \subsection{Continuous integration helpers}
 % Brigitta
 
 \astropy also provides a set of scripts for setting up and configuring
-continuous integration (CI) services as a \github repository,
+CI services as a \github repository,
 \package{astropy/ci-helpers}.\footnote{\url{https://github.com/astropy/ci-helpers}}
-These tools aim to empower package maintainers to control their testing set up
-and installation process for various continuous integration services through a
+These tools aim to empower package maintainers to control their testing setup
+and installation process for various CI services through a
 set of environment variables.
 While the current development is mostly driven by the needs of the \astropy
 ecosystem, the actual usage of this package is extremely widespread. The current
 tools support configuration for Travis CI and Appveyor CI.
 
 \subsection{Sphinx extensions}
 
-The documentation for many \python packages, the core \astropypkg package, and
-for all packages in the \astropy ecosystem is written using the
+The documentation for many \python packages, including
```

Comment:
Note: Isn't core package part of the ecosystem? If so, no need to be so repetitive.